### PR TITLE
 Frontend: ProductDetail Component #221

### LINKED ISSUE #76: Frontend: Product Detail Component
## **User story**
Who: As a developer
What: I want to develop reusable Product previewcomponent.
Why: So that display the details about each Image like, price ,name, size, and quantity of each of them.

## **Acceptance criteria**

This component should look according to screenshot below: 
Screen Shot 2022-02-03 at 10 31 39 PM

Check the wireframes here: https://www.figma.com/file/yo4QT2EcPo83HAMI3ydMxb/HYF-Wireframes?node-id=4%3A1324

## **Implementation details**

You will need to include in the Product details. Container the components and elements for:
Button - [issue#38](https://github.com/HackYourFuture-CPH/fp-class19/pull/38)
Counter - a counter that behaves like this example: [counter](https://codesandbox.io/s/74qzy90m3x)

### LINKED ISSUE #179: Frontend: ProductPage
## **User story**
Who: As a developer
What: I want to implement a ProductPage container
Why: Users should be able to see a single Product page with details of that product.

## **Acceptance criteria**
image

Check the wireframes here:
https://www.figma.com/file/yo4QT2EcPo83HAMI3ydMxb/HYF-Wireframes?node-id=216%3A28468

### LINKED ISSUE #221: Frontend: ProductDetail Component
## **User story**
Who: As a developer
What: I want to develop reusable Product detail component.
Why: So that display the details about each Image like, price ,name, size, and quantity of each of them.

## **Acceptance criteria**

This component should look according to screenshot below: 
Screen Shot 2022-02-03 at 10 31 39 PM

Check the wireframes here: https://www.figma.com/file/yo4QT2EcPo83HAMI3ydMxb/HYF-Wireframes?node-id=4%3A1324

## **Implementation details**

You will need to include in the Product details. Container the components and elements for:

Counter - a counter that behaves like this example: [counter](https://codesandbox.io/s/74qzy90m3x)

### PR DESCRIPTION
**Pull request by Impressiv3**
# Description
Frontend: ProductDetail Component

Fixes # (issue)
https://github.com/HackYourFuture-CPH/fp-class19/issues/221

# How to test?
npm run storybook

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by saloumeh-67 — APPROVED** [review]
it look the same as wireframes .Thanks

**Comment by Impressiv3** [comment]
Created a new Issue and a PR 
Issue: https://github.com/HackYourFuture-CPH/fp-class19/issues/226
PR: https://github.com/HackYourFuture-CPH/fp-class19/pull/227

File: src/client/components/ProductDetail/ProductDetail.stories.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import ProductDetail from './ProductDetail.component';
+
+export default {
+  title: 'Components / ProductDetail',
+  component: ProductDetail,
+};
+
+function Template(args) {
```
Comment (Divyajg) [inline src/client/components/ProductDetail/ProductDetail.stories.js:9]: I think we don't need "args" here.

File: src/client/components/ProductDetail/ProductDetail.styles.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+}
+
+.image-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.image-container img {
+  max-width: 280px;
```
Comment (Divyajg) [inline src/client/components/ProductDetail/ProductDetail.styles.css:38]: adding height: 230px; might make the classes image-container and info-container of equal height!

**Pull request closed without merging**